src/codeu/chat/DerbyStore.java

```diff
@@ -0,0 +1,283 @@
+package codeu.chat;
+
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Comparator;
+import java.util.HashSet;
+
+import codeu.chat.common.Conversation;
+import codeu.chat.common.Message;
+import codeu.chat.common.Time;
+import codeu.chat.common.User;
+//import codeu.chat.common.Uuid;
+import codeu.chat.util.store.Store;
+
+import codeu.chat.util.Uuid;
+
+
+import java.sql.Connection;
+
+public class DerbyStore {
+	
+	public String driver = "org.apache.derby.jdbc.EmbeddedDriver";
+	public String protocol = "jdbc:derby://localhost:1527/";
```

Comment:
Do you want to start up a new derby server, or could you just use the file protocol?